src/scaffold.ts

```diff
@@ -71,28 +74,37 @@ async function resourceScaffoldProjectRequest(
       return;
     }
     const bootstrapServers: string = removeProtocolPrefix(cluster.bootstrapServers);
-    return await scaffoldProjectRequest({
-      bootstrap_server: bootstrapServers,
-      cc_bootstrap_server: bootstrapServers,
-      cc_topic: item.name,
-      topic: item.name,
-      templateType: "kafka",
-    });
+    return await scaffoldProjectRequest(
+      {
+        bootstrap_server: bootstrapServers,
+        cc_bootstrap_server: bootstrapServers,
+        cc_topic: item.name,
+        topic: item.name,
+        templateType: "kafka",
+      },
+      "topic",
+    );
   } else if (item instanceof CCloudFlinkComputePool) {
     const organization: CCloudOrganization | undefined =
       await CCloudResourceLoader.getInstance().getOrganization();
-    return await scaffoldProjectRequest({
-      cc_environment_id: item.environmentId,
-      cc_organization_id: organization?.id,
-      cloud_region: item.region,
-      cloud_provider: item.provider,
-      cc_compute_pool_id: item.id,
-      templateType: "flink",
-    });
+    return await scaffoldProjectRequest(
+      {
+        cc_environment_id: item.environmentId,
+        cc_organization_id: organization?.id,
+        cloud_region: item.region,
+        cloud_provider: item.provider,
+        cc_compute_pool_id: item.id,
+        templateType: "flink",
+      },
+      "compute pool",
+    );
   }
 }
 
-export const scaffoldProjectRequest = async (templateRequestOptions?: PrefilledTemplateOptions) => {
+export const scaffoldProjectRequest = async (
+  templateRequestOptions?: PrefilledTemplateOptions,
+  telemetrySource?: string,
```

Comment:
This is mainly to provide `PostResponse` context back to the `CreateProjectTool` tool to feed back into the conversation, as well as allow callers to set their own `telemetrySource`.

cc @noeldevelops @Cerchie